site/content/docs/5.2/forms/checks-radios.md

```diff
@@ -250,6 +250,13 @@ Create button-like checkboxes and radio buttons by using `.btn` styles rather th
 
 ### Checkbox toggle buttons
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+These variants with only **one checkbox toggle button** should not be used because they do not respect the Orange Design System specifications. From the Orange Design System point of view and for usability reasons, a checkbox toggle button should not be used alone.
```

Comment:
When I read this callout, I don't understand why "a checkbox toggle button should not be used alone". A checkbox can be used alone, so why not a toggle button?

Reply:
I don't know how I give more explanation on this one. The reason is just that the design wants a that an isolated toggle button looks like a "real" checkbox and not like a button, mainly for usability reasons.

Reply:
Resolved since a checkbox behavior should look like a checkbox ?

Reply:
Yes, current version of the design callout message OK.